### ISSUE TITLE
Vertical rules w/ Grid

### ISSUE THREAD

**Issue by shaunrashid** [rachelandrew/cssgrid-ama#3]
With grid, is it possible to style the grid items so that they have a vertical rule on them except when the grid item is the last one on the row? Is there a way to make this responsive as well so it adapts as the screen size changes?

![grid-diagram](https://cloud.githubusercontent.com/assets/1373534/20042180/56526e46-a42a-11e6-9f5f-8c33eb3810b1.gif)

**Comment by Rothrock42** [comment]
You can create responsive grids by using repeat() with auto-fill and minmax(). Here is a basic example https://jsfiddle.net/Rothrock42/q06yej0e/

I'm not sure about how to do the vertical rule, but would be interested in how.

**Comment by rachelandrew** [comment]
You can't add borders to the grid cells or areas, but you can add them to the contents, like this: http://codepen.io/rachelandrew/pen/ObVjgg

**Comment by Rothrock42** [comment]
I've updated my fiddle to include a solution for having the vertical line not show at the end of a row or for the final item. https://jsfiddle.net/Rothrock42/q06yej0e/

I set the `overflow-x:hidden` on the grid container and made each grid item a little wider than the column it is in. That way the vertical line is in the gap, but on the last item in the row there is no gap and the overflow is hidden. Also used `nth-last-child(n+2)` to have no line on the last item.

It seems to work well in Chrome, but in Firefox I end up with a vertical scrollbar. But then again `grid-auto-rows:max-content` doesn't seem to be working yet either in FF....
